MongoDb_Test/QueryTest.js

```diff
@@ -0,0 +1,31 @@
+const MongoClient = require('mongodb').MongoClient;
+const url = "mongodb://localhost:27017";
+
+MongoClient.connect(url, function(err, client, db) {
+  if (err) throw err;
+  db = client.db('photos');
+
+  let querying = () => {
```

Comment:
minor note on function names - I think stylistically the standard is to try to use an imperative verb such as `query` or `queryDb`, etc. - not a big deal so no need to change depending on what you're aiming for, but in case you're curious check out some of these style guides and the examples they give for function/method names!
https://github.com/ryanmcdermott/clean-code-javascript#function-names-should-say-what-they-do
https://google.github.io/styleguide/jsguide.html#naming-method-names